### PR TITLE
small workshop fixes and updated versions

### PR DESCRIPTION
**Pull request by zackbradys**
*Issue #, if available:* N/A

*Description of changes:*
- Updated RKE2 to `v1.30.7`
- Updated Rancher to `v2.10.1`
- Fixed repository bug with updating the base image to `SP6`
- Fixed storage bug with updating to latest manifests on `master` branch
- Small fixes to workshop formatting/spelling/roadmap/contact information
- Tested and verified comparability with two accounts in `us-east-1` and one in `us-west-2`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ This document describes the current status and the upcoming milestones of the Ra
 |   ✔    | **[Determine Improvements and Launch Roadmap](#determine-improvements-and-launch-roadmap)** | 2 / 2 | `enhancement` | Jul 21 2023 |
 |   ✔    |           **[Improve Screenshot Highlighting](#improve-screenshot-highlighting)**           | 6 / 6 | `enhancement` | Sep 01 2023 |
 |   ✔    |  **[Cleanup Workshop Formatting and Language](#cleanup-workshop-formatting-and-language)**  | 2 / 2 | `enhancement` | Sep 01 2023 |
-|   🚀   |          **[Add Additional Demo Applications](#add-additional-demo-applications)**          | 0 / 3 |   `feature`   | Apr 01 2024 |
-|   🚀   |  **[Implement Support for Additional Regions](#implement-support-for-additional-regions)**  | 0 / 4 |   `feature`   | Jun 01 2024 |
-|   🚀   |      **[Cloud9 Optimization and Improvements](#cloud9-optimization-and-improvements)**      | 0 / 4 | `enhancement` | May 01 2024 |
+|   🚀   |          **[Add Additional Demo Applications](#add-additional-demo-applications)**          | 0 / 3 |   `feature`   | Jan 31 2024 |
```
Comment (evdevr) [inline ROADMAP.md:12]: should this now be 2025 for future planned work?

Comment (zackbradys) [reply ROADMAP.md:12]: back to the future! 😄 ... thanks for catching it and I'll update now

**Comment by zackbradys** [comment]
- Fixed `roadmap` dates for `2025`
- Updated Rancher from `v2.10.0` to `v2.10.1`
- Tested and verified `ami` in `us-east-1` and `us-west-2`

**Review by evdevr — APPROVED** [review]
LGTM

**Pull request merged**